libs/datamodel/core/src/transform/ast_to_dml/validate.rs

```diff
@@ -506,36 +502,6 @@ impl<'a> Validator<'a> {
         }
     }
 
-    /// Ensures that embedded types do not have back relations
-    /// to their parent types.
-    fn validate_embedded_types_have_no_back_relation(
-        &self,
-        ast_schema: &ast::SchemaAst,
-        datamodel: &dml::Datamodel,
-        model: &dml::Model,
-    ) -> Result<(), DatamodelError> {
-        if model.is_embedded {
```

Comment:
Wow, I didn't know we had `model.is_embedded`. Not sure why that code still existed, we might want to clean up further in the not too distant future.

Reply:
Yeah, Mongo remnants